arrow-fx-test/src/main/kotlin/arrow/test/eq/EqK.kt

```diff
@@ -0,0 +1,53 @@
+package arrow.test.eq
```

Comment:
Hi! Please, if `arrow-fx-test` is going to be publish in OSS repository, replace `arrow.test` package by `arrow.fx.test` in order to avoid misunderstandings with other test artifacts. We'll do the same with `arrow-core-test` that is already published.

Reply:
Thanks!! :raised_hands: